logic/auth/login.go

```
@@ -0,0 +1,144 @@
+
+type LoginLogic struct {
+	logic.Base
+
+	captcha *commonlogic.CaptchaLogic
+	token   TokenLogic
+}
+
+// Login 登录
+func (l *LoginLogic) Login(ctx *gin.Context, req *authtype.LoginReq) (*authtype.TokenRes, error) {
+	var (
+		res = &authtype.TokenRes{}
```

golangci-lint

ineffectual assignment to res

(ineffassign)